Source/Views/SPAlertSheets.m

```diff
@@ -1,189 +0,0 @@
-//  FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
-//  OTHER DEALINGS IN THE SOFTWARE.
-//
-//  More info at <https://github.com/sequelpro/sequelpro>
-
-#import "SPAlertSheets.h"
-#import "SPMainThreadTrampoline.h"
-#import "SPFunctions.h"
-
-@implementation SPAlertSheets
-
-/**
- * Provide a simple alias of a NSApp-wide modal NSBeginAlertSheet which waits 
- * for a in calling class globally declared returnCode by reference which must 
- * be changed in the didEndSelector of the calling class, with a few differences:
- *  - printf-type format strings are no longer supported within the "msg"
- *    message text argument, preventing access of random stack areas for
- *    error text which contains inadvertant printf formatting.
- *  - The didDismissSelector is no longer supported
- *  - The sheet no longer needs to be orderOut:ed after use
- *  - The alert is always shown on the main thread.
- */
-+ (void)beginWaitingAlertSheetWithTitle:(NSString *)title
-                          defaultButton:(NSString *)defaultButton
-                        alternateButton:(NSString *)alternateButton
-                            otherButton:(NSString *)otherButton
-                             alertStyle:(NSAlertStyle)alertStyle
-                              docWindow:(NSWindow *)docWindow
-                          modalDelegate:(id)modalDelegate
-                         didEndSelector:(SEL)didEndSelector
-                            contextInfo:(void *)contextInfo
-                               infoText:(NSString *)infoText
-                             returnCode:(NSInteger *)returnCode
-{
-
-	// Ensure execution on the main thread
-	if (![[NSThread currentThread] isMainThread]) {
-		return [[self onMainThread] beginWaitingAlertSheetWithTitle:title
-		                                              defaultButton:defaultButton
-		                                            alternateButton:alternateButton
-		                                                otherButton:otherButton
-		                                                 alertStyle:alertStyle
-		                                                  docWindow:docWindow
-		                                              modalDelegate:modalDelegate
-		                                             didEndSelector:didEndSelector
-		                                                contextInfo:contextInfo
-		                                                   infoText:infoText
-		                                                 returnCode:returnCode];
-	}
-
-	NSButton *aButton;
-
-	// Initialize returnCode with a value which can't be returned as
-	// returnCode in the didEndSelector method
-	NSInteger initialReturnCode = -5;
-	returnCode = &initialReturnCode;
-
-	// Set up an NSAlert with the supplied details
-	NSAlert *alert = [[NSAlert alloc] init];
-	[alert setMessageText:title];
-
-	aButton = [alert addButtonWithTitle:defaultButton];
-	[aButton setTag:NSAlertDefaultReturn];
```

Comment:
@Sequel-Ace/all This explains why in rare cases you saw `NSAlertDefaultReturn` instead of modern correct one `NSAlertFirstRuturn` - by removing this, you can now ALWAYS use the modern enum. :)